aws-lambda-java-runtime-interface-client/src/main/java/com/amazonaws/services/lambda/runtime/api/client/runtimeapi/NativeClient.java

```diff
@@ -44,10 +48,25 @@
             initializeClient(userAgent.getBytes());
     }
 
+    /**
+     * @return a string describing the detected architecture the RIC is executing on
+     * @throws RuntimeException
```

Comment:
`RuntimeException` -> `UnknownPlatformException`

Reply:
incorporated in the last update